### ISSUE TITLE
Ember input hatch not providing FE to machine

### ISSUE THREAD

**Issue by Febilian** [DaedalusGame/Modular-Diversity#17]
I've tried to make a machine that consumes ember and optionally alchemical slurry to generate FE. In creative mode, I built the structure properly and made sure there was an ember input hatch(tried tiny and small sizes), a fluid input hatch, and an energy output hatch on the machine. The structure checks out, and the recipes show up in JEI, but no matter how much ember I send into the ember hatch with a creative ember source, the machine doesn't seem to understand that it's got any ember to turn into FE.

I apologize if I've gravely misunderstood the wiki page, which seems to indicate that the internal values of the ember input hatches uses the same format(with different number values that convert to FE automatically) as normal energy inputs. I've really tried all I can to figure this out on my own and at this point I figure I'm either not noticing something idiotic, the documentation is unclear to the point I would not have figured it out on my own, or it's not working.

Here are pictures of what I'm looking at for clarity.

https://imgur.com/a/oUN6hZ2

Machine JSON file contents: https://gist.github.com/Febilian/565585c3cd66044a6f5d81a2615ee9b9

The variables used in the machine file: https://gist.github.com/Febilian/1eaedde75b512c45033b367b66cc3b5f

My two recipes for the machine:

https://gist.github.com/Febilian/aa8be9720997ad179e39932a8e03f20a

https://gist.github.com/Febilian/ae199507eaef397a7890faf8a8b413d8

I'm using Modular Machinery version 1.9.4, and Modular Diversity 1.0.

Any help knowing what I did wrong(if I did something wrong) would be appreciated.

**Comment by DaedalusGame** [comment]
right, this stuff changed actually. Ember, Mana, BWM Mechpower and BC Lasers are now their own thing, they no longer convert straight to FE/RF internally. I'll have to update the docs when I wake up, but for now here's examples i use in dev:

```
{
    "type": "ember",
    "io-type": "output",
    "ember": 16
}
```
```
{
    "type": "mana",
    "io-type": "output",
    "mana": 50
}
```

```
{
    "type": "mechanical",
    "io-type": "input",
    "level": 1,
    "crank-allowed": true
}
```
```
{
    "type": "laser",
    "io-type": "input",
    "micro-mj": 30000000
}
```
(Side note here, Laser is per tick, and this means 30 MJ/t)
(Ember and Mana are consume-once, but I'm planning per-tick versions)

**Comment by Febilian** [comment]
Got it working. Thanks!
